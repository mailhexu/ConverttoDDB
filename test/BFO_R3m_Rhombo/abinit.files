abinit.in
abinit.out
abinit-i
abinit-o
abinit-t
Bi.psp
Fe.psp
O.psp
