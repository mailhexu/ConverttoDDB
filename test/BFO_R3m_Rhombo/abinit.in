natom 10
ntypat 3
znucl  83 26 8
typat  2*1 2*2 6*3
acell  10.75248368 10.75248368 10.75248368
rprim  0.56543862 -0.28271931 -0.28271931
 0.00000000 0.48968421 -0.48968421
 0.82479038 0.82479038 0.82479038
xred 
0.28930873  0.28930873  0.28930873
0.78930873  0.78930873  0.78930873
0.00316666  0.00316666  0.00316666
0.50316666  0.50316666  0.50316666
0.21872786  0.74489837  0.74489837
0.74489837  0.21872786  0.74489837
0.74489837  0.74489837  0.21872786
0.24489837  0.71872786  0.24489837
0.24489837  0.24489837  0.71872786
0.71872786  0.24489837  0.24489837

ecut   10
kptopt 1
ngkpt  1 1 1
ixc    1
nstep  1
toldfe 1.0E+05
iscf   5
prtden 0
prtwf  0
prteig 0
 chkprim 0
