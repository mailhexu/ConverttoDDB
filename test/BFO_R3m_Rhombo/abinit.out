
.Version 8.4.2 of ABINIT 
.(MPI version, prepared for a x86_64_linux_gnu5.4 computer) 

.Copyright (C) 1998-2017 ABINIT group . 
 ABINIT comes with ABSOLUTELY NO WARRANTY.
 It is free software, and you are welcome to redistribute it
 under certain conditions (GNU General Public License,
 see ~abinit/COPYING or http://www.gnu.org/copyleft/gpl.txt).

 ABINIT is a project of the Universite Catholique de Louvain,
 Corning Inc. and other collaborators, see ~abinit/doc/developers/contributors.txt .
 Please read ~abinit/doc/users/acknowledgments.html for suggested
 acknowledgments of the ABINIT effort.
 For more information, see http://www.abinit.org .

.Starting date : Tue 18 Jul 2017.
- ( at 10h10 )
  
- input  file    -> abinit.in
- output file    -> abinit.out
- root for input  files -> abinit-i
- root for output files -> abinit-o


 Symmetries : the unit cell is not primitive
================================================================================
 Values of the parameters that define the memory need of the present run
     intxc =       0    ionmov =       0      iscf =       5    lmnmax =       6
     lnmax =       6     mgfft =      45  mpssoang =       3    mqgrid =    3001
     natom =      10  nloc_mem =       1    nspden =       1   nspinor =       1
    nsppol =       1      nsym =       4    n1xccc =       0    ntypat =       3
    occopt =       1   xclevel =       1
-    mband =          41        mffmem =           1         mkmem =           1
       mpw =         652          nfft =       25920          nkpt =           1
================================================================================
P This job should need less than                       7.939 Mbytes of memory.
  Rough estimation (10% accuracy) of disk space for files :
_ WF disk file :      0.410 Mbytes ; DEN or POT disk file :      0.200 Mbytes.
================================================================================

--------------------------------------------------------------------------------
------------- Echo of variables that govern the present computation ------------
--------------------------------------------------------------------------------
-
- outvars: echo of selected default values                                      
-   iomode0 =  0 , fftalg0 =112 , wfoptalg0 =  0
-
- outvars: echo of global parameters not present in the input file              
-  max_nthreads =    0
-
 -outvars: echo values of preprocessed input variables --------
            acell      1.0752483680E+01  1.0752483680E+01  1.0752483680E+01 Bohr
              amu      2.08980370E+02  5.58470000E+01  1.59994000E+01
             ecut      1.00000000E+01 Hartree
-          fftalg         112
             iscf           5
           istwfk        9
              kpt      5.00000000E-01  5.00000000E-01  5.00000000E-01
         kptrlatt        1    0    0      0    1    0      0    0    1
          kptrlen      7.44628903E+00
P           mkmem           1
            natom          10
            nband          41
            ngfft          24      24      45
             nkpt           1
            nstep           1
             nsym           4
           ntypat           3
              occ      2.000000  2.000000  2.000000  2.000000  2.000000  2.000000
                       2.000000  2.000000  2.000000  2.000000  2.000000  2.000000
                       2.000000  2.000000  2.000000  2.000000  2.000000  2.000000
                       2.000000  2.000000  2.000000  2.000000  2.000000  2.000000
                       2.000000  2.000000  2.000000  2.000000  2.000000  2.000000
                       2.000000  2.000000  2.000000  2.000000  2.000000  2.000000
                       2.000000  2.000000  2.000000  0.000000  0.000000
           prtden           0
           prteig           0
            prtwf           0
            rprim      5.6543862000E-01 -2.8271931000E-01 -2.8271931000E-01
                       0.0000000000E+00  4.8968421000E-01 -4.8968421000E-01
                       8.2479038000E-01  8.2479038000E-01  8.2479038000E-01
           shiftk      5.00000000E-01  5.00000000E-01  5.00000000E-01
           symrel      1  0  0   0  1  0   0  0  1       1  0  0   0  1  0   0  0  1
                       0  1  0   1  0  0   0  0  1       0  1  0   1  0  0   0  0  1
            tnons      0.0000000  0.0000000  0.0000000     0.5000000  0.5000000  0.5000000
                       0.0000000 -0.0000000  0.0000000     0.5000000  0.5000000  0.5000000
           toldfe      1.00000000E+05 Hartree
            typat      1  1  2  2  3  3  3  3  3  3
           xangst      2.2885363006E+00  1.6984319933E+00  8.6237037771E-02
                       6.2437164650E+00  4.6337599270E+00  2.3527685031E-01
                       2.5049421639E-02  1.8590371109E-02  9.4391682556E-04
                       3.9802295861E+00  2.9539183048E+00  1.4998372937E-01
                       4.1995511961E+00  5.2194736460E+00  1.0684706863E+00
                       5.8924145151E+00  2.9069793448E+00  1.6881016686E+00
                       3.4230795055E+00  1.9037069770E+00 -2.2472959827E+00
                       1.9372343506E+00  2.7579395322E+00 -1.2472262651E+00
                       4.1609312816E+00  3.6614109837E+00  2.2966961452E+00
                       3.4616994201E+00  6.7548151806E-01 -6.8923332040E-01
            xcart      4.3247068533E+00  3.2095713226E+00  1.6296438390E-01
                       1.1798914170E+01  8.7565372276E+00  4.4460881250E-01
                       4.7336546685E-02  3.5130710105E-02  1.7837442925E-03
                       7.5215438637E+00  5.5820966151E+00  2.8342817289E-01
                       7.9360016417E+00  9.8633757487E+00  2.0191169782E+00
                       1.1135049695E+01  5.4933948356E+00  3.1900498381E+00
                       6.4686827965E+00  3.5974848237E+00 -4.2467739469E+00
                       3.6608423779E+00  5.2117504070E+00 -2.3569160669E+00
                       7.8630205799E+00  6.9190640191E+00  4.3401267249E+00
                       6.5416638583E+00  1.2764750770E+00 -1.3024622172E+00
             xred      2.8930873000E-01  2.8930873000E-01  2.8930873000E-01
                       7.8930873000E-01  7.8930873000E-01  7.8930873000E-01
                       3.1666600000E-03  3.1666600000E-03  3.1666600000E-03
                       5.0316666000E-01  5.0316666000E-01  5.0316666000E-01
                       2.1872786000E-01  7.4489837000E-01  7.4489837000E-01
                       7.4489837000E-01  2.1872786000E-01  7.4489837000E-01
                       7.4489837000E-01  7.4489837000E-01  2.1872786000E-01
                       2.4489837000E-01  7.1872786000E-01  2.4489837000E-01
                       2.4489837000E-01  2.4489837000E-01  7.1872786000E-01
                       7.1872786000E-01  2.4489837000E-01  2.4489837000E-01
            znucl       83.00000   26.00000    8.00000

================================================================================

 chkinp: Checking input parameters for consistency.

================================================================================
== DATASET  1 ==================================================================
-   nproc =    1

 Exchange-correlation functional for the present dataset will be:
  LDA: new Teter (4/93) with spin-polarized option - ixc=1
 Citation for XC functional:
  S. Goedecker, M. Teter, J. Huetter, PRB 54, 1703 (1996)

 Real(R)+Recip(G) space primitive vectors, cartesian coordinates (Bohr,Bohr^-1):
 R(1)=  6.0798695 -3.0399348 -3.0399348  G(1)=  0.1096515 -0.0548257 -0.0548257
 R(2)=  0.0000000  5.2653215 -5.2653215  G(2)=  0.0000000  0.0949610 -0.0949610
 R(3)=  8.8685451  8.8685451  8.8685451  G(3)=  0.0375860  0.0375860  0.0375860
 Unit cell volume ucvol=  8.5171204E+02 bohr^3
 Angles (23,13,12)=  9.00000000E+01  9.00000000E+01  9.00000000E+01 degrees

 getcut: wavevector=  0.0000  0.0000  0.0000  ngfft=  24  24  45
         ecut(hartree)=     10.000   => boxcut(ratio)=   2.01222

--- Pseudopotential description ------------------------------------------------
- pspini: atom type   1  psp file is Bi.psp
- pspatm: opening atomic psp file    Bi.psp
- Goedecker pseudopotential for Bi
- 83.00000   5.00000     70301                znucl, zion, pspdat
   10    1    2    0      2001   0.00000      pspcod,pspxc,lmax,lloc,mmax,r2well
 rloc=   0.6050000
 cc(1:1)=   6.6794371
 for angular momentum l =  0 r(l) =   0.6788580
   h11, h12, h13 =   1.3776338   0.1989540  -0.1149191
        h22, h23 =              -0.5136969   0.2967198
             h33 =                          -0.4710281
 for angular momentum l =  1 r(l) =   0.7986727
   h11, h12, h13 =   0.6555781   0.1702699   0.0000000
        h22, h23 =              -0.4029322   0.0000000
             h33 =                           0.0000000
   k11, k12, k13 =   0.3053135   0.0097757   0.0000000
        k22, k23 =              -0.0231337   0.0000000
             k33 =                           0.0000000
 for angular momentum l =  2 r(l) =   0.9346826
   h11, h12, h13 =   0.3784758   0.0000000   0.0000000
        h22, h23 =               0.0000000   0.0000000
             h33 =                           0.0000000
   k11, k12, k13 =   0.0292174   0.0000000   0.0000000
        k22, k23 =               0.0000000   0.0000000
             k33 =                           0.0000000
-  Local part computed in reciprocal space.

 pspatm : COMMENT -
  the projectors are not normalized,
  so that the KB energies are not consistent with 
  definition in PRB44, 8503 (1991). 
  However, this does not influence the results obtained hereafter.
 pspatm: epsatm=   34.79471605
         --- l  ekb(1:nproj) -->
             0   -0.901401   -0.217891    1.555038
             1   -0.989899    1.571991
             2    3.344660
 pspatm: atomic psp has been read  and splines computed

- pspini: atom type   2  psp file is Fe.psp
- pspatm: opening atomic psp file    Fe.psp
- Goedecker pseudopotential for Fe
- 26.00000  16.00000     70301                znucl, zion, pspdat
   10    1    2    2      2001   0.00000      pspcod,pspxc,lmax,lloc,mmax,r2well
 rloc=   0.3600000
 cc(1:2)=   5.3925067  -0.0300664
 for angular momentum l =  0 r(l) =   0.2692681
   h11, h12, h13 =  10.1937228   2.6471772   0.0000000
        h22, h23 =              -6.8349821   0.0000000
             h33 =                           0.0000000
 for angular momentum l =  1 r(l) =   0.2476856
   h11, h12, h13 =   0.1456126   2.2121720   0.0000000
        h22, h23 =              -5.2349543   0.0000000
             h33 =                           0.0000000
   k11, k12, k13 =   0.2014502   0.0320434   0.0000000
        k22, k23 =              -0.0758286   0.0000000
             k33 =                           0.0000000
 for angular momentum l =  2 r(l) =   0.2230210
   h11, h12, h13 = -12.0269410   0.0000000   0.0000000
        h22, h23 =               0.0000000   0.0000000
             h33 =                           0.0000000
   k11, k12, k13 =   0.0103223   0.0000000   0.0000000
        k22, k23 =               0.0000000   0.0000000
             k33 =                           0.0000000
-  Local part computed in reciprocal space.

 pspatm : COMMENT -
  the projectors are not normalized,
  so that the KB energies are not consistent with 
  definition in PRB44, 8503 (1991). 
  However, this does not influence the results obtained hereafter.
 pspatm: epsatm=   16.92502181
         --- l  ekb(1:nproj) -->
             0   -0.500862    0.733316
             1   -0.039837    0.006202
             2   -0.004680
 pspatm: atomic psp has been read  and splines computed

- pspini: atom type   3  psp file is O.psp
- pspatm: opening atomic psp file    O.psp
- Goedecker pseudopotential for O
-  8.00000   6.00000     70301                znucl, zion, pspdat
   10    1    1    2      2001   0.00000      pspcod,pspxc,lmax,lloc,mmax,r2well
 rloc=   0.2476209
 cc(1:2)= -16.5803180   2.3957009
 for angular momentum l =  0 r(l) =   0.2217861
   h11, h12, h13 =  18.2669172   0.0000000   0.0000000
        h22, h23 =               0.0000000   0.0000000
             h33 =                           0.0000000
 for angular momentum l =  1 r(l) =   0.2568289
   h11, h12, h13 =   0.0000000   0.0000000   0.0000000
        h22, h23 =               0.0000000   0.0000000
             h33 =                           0.0000000
   k11, k12, k13 =   0.0044758   0.0000000   0.0000000
        k22, k23 =               0.0000000   0.0000000
             k33 =                           0.0000000
-  Local part computed in reciprocal space.

 pspatm : COMMENT -
  the projectors are not normalized,
  so that the KB energies are not consistent with 
  definition in PRB44, 8503 (1991). 
  However, this does not influence the results obtained hereafter.
 pspatm: epsatm=    0.06537593
         --- l  ekb(1:nproj) -->
             0    0.706437
             1    0.000000
 pspatm: atomic psp has been read  and splines computed

   8.09887504E+03                                ecore*ucvol(ha*bohr**3)
--------------------------------------------------------------------------------

_setup2: Arith. and geom. avg. npw (full set) are    1304.000    1304.000

================================================================================

     iter   Etot(hartree)      deltaE(h)  residm     vres2
 ETOT  1  -317.21122286014    -3.172E+02 5.415E-02 2.059E+03

 Cartesian components of stress tensor (hartree/bohr^3)
  sigma(1 1)=  2.50189244E-02  sigma(3 2)=  2.13317190E-03
  sigma(2 2)=  2.38304263E-02  sigma(3 1)=  2.04308010E-03
  sigma(3 3)=  2.27422793E-02  sigma(2 1)=  3.95583520E-03

 Integrated electronic density in atomic spheres:
 ------------------------------------------------
 Atom  Sphere_radius  Integrated_density
    1        2.00000          1.94874581
    2        2.00000          1.99706431
    3        2.00000         13.04125109
    4        2.00000         13.03731305
    5        2.00000          5.43352832
    6        2.00000          5.43352832
    7        2.00000          4.27348026
    8        2.00000          5.42737991
    9        2.00000          4.29914240
   10        2.00000          5.42737991
================================================================================

 ----iterations are completed or convergence reached----

 Mean square residual over all n,k,spin=   1.2880E-02; max=  5.4148E-02
 reduced coordinates (array xred) for   10 atoms
       0.289308730000      0.289308730000      0.289308730000
       0.789308730000      0.789308730000      0.789308730000
       0.003166660000      0.003166660000      0.003166660000
       0.503166660000      0.503166660000      0.503166660000
       0.218727860000      0.744898370000      0.744898370000
       0.744898370000      0.218727860000      0.744898370000
       0.744898370000      0.744898370000      0.218727860000
       0.244898370000      0.718727860000      0.244898370000
       0.244898370000      0.244898370000      0.718727860000
       0.718727860000      0.244898370000      0.244898370000
 rms dE/dt=  1.8648E+00; max dE/dt=  2.7419E+00; dE/dt below (all hartree)
    1       2.539397487030      2.539397487030      2.741872170795
    2       2.539397487030      2.539397487030      2.741872170795
    3      -1.197202686914     -1.197202686914     -4.195652423972
    4      -1.197202686914     -1.197202686914     -4.195652423972
    5      -1.781106098986      0.436235282346     -0.028817956794
    6       0.436235282346     -1.781106098986     -0.028817956794
    7       0.331208500399      0.331208500399      2.081009474674
    8       0.436235282346     -1.781106098986     -0.028817956794
    9       0.331208500399      0.331208500399      2.081009474674
   10      -1.781106098986      0.436235282346     -0.028817956794

 cartesian coordinates (angstrom) at end:
    1      2.28853630058090     1.69843199328745     0.08623703777108
    2      6.24371646500405     4.63375992702704     0.23527685031161
    3      0.02504942163894     0.01859037110931     0.00094391682556
    4      3.98022958606209     2.95391830484890     0.14998372936609
    5      4.19955119612023     5.21947364599119     1.06847068632812
    6      5.89241451507027     2.90697934477792     1.68810166859134
    7      3.42307950550749     1.90370697695339    -2.24729598270969
    8      1.93723435064712     2.75793953223739    -1.24722626514824
    9      4.16093128155319     3.66141098368265     2.29669614521863
   10      3.46169942007453     0.67548151806288    -0.68923332040114

 cartesian forces (hartree/bohr) at end:
    1     -0.37001818663628    -0.19805646481093     0.27175169435883
    2     -0.37001818663628    -0.19805646481093     0.27175169435883
    3      0.30045949036169     0.21266677955353    -0.02718736598237
    4      0.30045949036169     0.21266677955353    -0.02718736598237
    5      0.20787062825293    -0.13107373200317    -0.06070219111370
    6     -0.03526412643012     0.20105451909312    -0.14969568752695
    7     -0.10304780554822    -0.08459110183255    -0.03416644973581
    8     -0.03526412643012     0.20105451909312    -0.14969568752695
    9     -0.10304780554822    -0.08459110183255    -0.03416644973581
   10      0.20787062825293    -0.13107373200317    -0.06070219111370
 frms,max,avg= 1.8856006E-01 3.7001819E-01  -1.149E-02 -6.919E-03  5.560E-03 h/b

 cartesian forces (eV/Angstrom) at end:
    1    -19.02709857548323   -10.18447204913610    13.97403279991377
    2    -19.02709857548323   -10.18447204913610    13.97403279991377
    3     15.45024690008241    10.93576457708865    -1.39803045157556
    4     15.45024690008241    10.93576457708865    -1.39803045157556
    5     10.68913658182941    -6.74008172990789    -3.12143190735517
    6     -1.81335413770806    10.33863818588861    -7.69766110361420
    7     -5.29893076872053    -4.34984898393328    -1.75690933736885
    8     -1.81335413770806    10.33863818588861    -7.69766110361419
    9     -5.29893076872053    -4.34984898393328    -1.75690933736885
   10     10.68913658182941    -6.74008172990789    -3.12143190735517
 frms,max,avg= 9.6961472E+00 1.9027099E+01  -5.907E-01 -3.558E-01  2.859E-01 e/A
 length scales= 10.752483680000 10.752483680000 10.752483680000 bohr
              =  5.689969299192  5.689969299192  5.689969299192 angstroms
 Fermi (or HOMO) energy (hartree) =   0.40175   Average Vxc (hartree)=  -0.44128
 Eigenvalues (hartree) for nkpt=   1  k points:
 kpt#   1, nband= 41, wtk=  1.00000, kpt=  0.5000  0.5000  0.5000 (reduced coord)
  -4.55803   -4.55785   -2.88892   -2.88863   -2.88096   -2.88027   -2.85926   -2.85887
  -0.43614   -0.43586   -0.39092   -0.39013   -0.28881   -0.28767   -0.19029   -0.18916
   0.06098    0.06217    0.16730    0.16860    0.17723    0.17931    0.20471    0.20721
   0.29696    0.30242    0.30649    0.31749    0.31863    0.32758    0.33396    0.33816
   0.34594    0.35420    0.35988    0.36263    0.37879    0.39093    0.40175    0.42786
   0.43326

--------------------------------------------------------------------------------
 Components of total free energy (in Hartree) :

    Kinetic energy  =  1.21094085144355E+02
    Hartree energy  =  7.50985302938279E+01
    XC energy       = -4.09236423231773E+01
    Ewald energy    = -2.11593484614085E+02
    PspCore energy  =  9.50893573350405E+00
    Loc. psp. energy= -2.82133797425138E+02
    NL   psp  energy=  1.17381503305735E+01
    >>>>>>>>> Etotal= -3.17211222860139E+02

 Other information on the energy :
    Total energy(eV)= -8.63175634994726E+03 ; Band energy (Ha)=  -4.5236913969E+01
--------------------------------------------------------------------------------

 Cartesian components of stress tensor (hartree/bohr^3)
  sigma(1 1)=  2.50189244E-02  sigma(3 2)=  2.13317190E-03
  sigma(2 2)=  2.38304263E-02  sigma(3 1)=  2.04308010E-03
  sigma(3 3)=  2.27422793E-02  sigma(2 1)=  3.95583520E-03

-Cartesian components of stress tensor (GPa)         [Pressure= -7.0210E+02 GPa]
- sigma(1 1)=  7.36082046E+02  sigma(3 2)=  6.27600736E+01
- sigma(2 2)=  7.01115228E+02  sigma(3 1)=  6.01094818E+01
- sigma(3 3)=  6.69100845E+02  sigma(2 1)=  1.16384670E+02

== END DATASET(S) ==============================================================
================================================================================
  
 -outvars: echo values of variables after computation  --------
            acell      1.0752483680E+01  1.0752483680E+01  1.0752483680E+01 Bohr
              amu      2.08980370E+02  5.58470000E+01  1.59994000E+01
             ecut      1.00000000E+01 Hartree
           etotal     -3.1721122286E+02
            fcart     -3.7001818664E-01 -1.9805646481E-01  2.7175169436E-01
                      -3.7001818664E-01 -1.9805646481E-01  2.7175169436E-01
                       3.0045949036E-01  2.1266677955E-01 -2.7187365982E-02
                       3.0045949036E-01  2.1266677955E-01 -2.7187365982E-02
                       2.0787062825E-01 -1.3107373200E-01 -6.0702191114E-02
                      -3.5264126430E-02  2.0105451909E-01 -1.4969568753E-01
                      -1.0304780555E-01 -8.4591101833E-02 -3.4166449736E-02
                      -3.5264126430E-02  2.0105451909E-01 -1.4969568753E-01
                      -1.0304780555E-01 -8.4591101833E-02 -3.4166449736E-02
                       2.0787062825E-01 -1.3107373200E-01 -6.0702191114E-02
-          fftalg         112
             iscf           5
           istwfk        9
              kpt      5.00000000E-01  5.00000000E-01  5.00000000E-01
         kptrlatt        1    0    0      0    1    0      0    0    1
          kptrlen      7.44628903E+00
P           mkmem           1
            natom          10
            nband          41
            ngfft          24      24      45
             nkpt           1
            nstep           1
             nsym           4
           ntypat           3
              occ      2.000000  2.000000  2.000000  2.000000  2.000000  2.000000
                       2.000000  2.000000  2.000000  2.000000  2.000000  2.000000
                       2.000000  2.000000  2.000000  2.000000  2.000000  2.000000
                       2.000000  2.000000  2.000000  2.000000  2.000000  2.000000
                       2.000000  2.000000  2.000000  2.000000  2.000000  2.000000
                       2.000000  2.000000  2.000000  2.000000  2.000000  2.000000
                       2.000000  2.000000  2.000000  0.000000  0.000000
           prtden           0
           prteig           0
            prtwf           0
            rprim      5.6543862000E-01 -2.8271931000E-01 -2.8271931000E-01
                       0.0000000000E+00  4.8968421000E-01 -4.8968421000E-01
                       8.2479038000E-01  8.2479038000E-01  8.2479038000E-01
           shiftk      5.00000000E-01  5.00000000E-01  5.00000000E-01
           strten      2.5018924440E-02  2.3830426255E-02  2.2742279312E-02
                       2.1331719033E-03  2.0430801036E-03  3.9558352013E-03
           symrel      1  0  0   0  1  0   0  0  1       1  0  0   0  1  0   0  0  1
                       0  1  0   1  0  0   0  0  1       0  1  0   1  0  0   0  0  1
            tnons      0.0000000  0.0000000  0.0000000     0.5000000  0.5000000  0.5000000
                       0.0000000 -0.0000000  0.0000000     0.5000000  0.5000000  0.5000000
           toldfe      1.00000000E+05 Hartree
            typat      1  1  2  2  3  3  3  3  3  3
           xangst      2.2885363006E+00  1.6984319933E+00  8.6237037771E-02
                       6.2437164650E+00  4.6337599270E+00  2.3527685031E-01
                       2.5049421639E-02  1.8590371109E-02  9.4391682556E-04
                       3.9802295861E+00  2.9539183048E+00  1.4998372937E-01
                       4.1995511961E+00  5.2194736460E+00  1.0684706863E+00
                       5.8924145151E+00  2.9069793448E+00  1.6881016686E+00
                       3.4230795055E+00  1.9037069770E+00 -2.2472959827E+00
                       1.9372343506E+00  2.7579395322E+00 -1.2472262651E+00
                       4.1609312816E+00  3.6614109837E+00  2.2966961452E+00
                       3.4616994201E+00  6.7548151806E-01 -6.8923332040E-01
            xcart      4.3247068533E+00  3.2095713226E+00  1.6296438390E-01
                       1.1798914170E+01  8.7565372276E+00  4.4460881250E-01
                       4.7336546685E-02  3.5130710105E-02  1.7837442925E-03
                       7.5215438637E+00  5.5820966151E+00  2.8342817289E-01
                       7.9360016417E+00  9.8633757487E+00  2.0191169782E+00
                       1.1135049695E+01  5.4933948356E+00  3.1900498381E+00
                       6.4686827965E+00  3.5974848237E+00 -4.2467739469E+00
                       3.6608423779E+00  5.2117504070E+00 -2.3569160669E+00
                       7.8630205799E+00  6.9190640191E+00  4.3401267249E+00
                       6.5416638583E+00  1.2764750770E+00 -1.3024622172E+00
             xred      2.8930873000E-01  2.8930873000E-01  2.8930873000E-01
                       7.8930873000E-01  7.8930873000E-01  7.8930873000E-01
                       3.1666600000E-03  3.1666600000E-03  3.1666600000E-03
                       5.0316666000E-01  5.0316666000E-01  5.0316666000E-01
                       2.1872786000E-01  7.4489837000E-01  7.4489837000E-01
                       7.4489837000E-01  2.1872786000E-01  7.4489837000E-01
                       7.4489837000E-01  7.4489837000E-01  2.1872786000E-01
                       2.4489837000E-01  7.1872786000E-01  2.4489837000E-01
                       2.4489837000E-01  2.4489837000E-01  7.1872786000E-01
                       7.1872786000E-01  2.4489837000E-01  2.4489837000E-01
            znucl       83.00000   26.00000    8.00000

================================================================================


- Timing analysis has been suppressed with timopt=0



================================================================================

 Suggested references for the acknowledgment of ABINIT usage.

 The users of ABINIT have little formal obligations with respect to the ABINIT group
 (those specified in the GNU General Public License, http://www.gnu.org/copyleft/gpl.txt).
 However, it is common practice in the scientific literature,
 to acknowledge the efforts of people that have made the research possible.
 In this spirit, please find below suggested citations of work written by ABINIT developers,
 corresponding to implementations inside of ABINIT that you have used in the present run.
 Note also that it will be of great value to readers of publications presenting these results,
 to read papers enabling them to understand the theoretical formalism and details
 of the ABINIT implementation.
 For information on why they are suggested, see also http://www.abinit.org/about/?text=acknowledgments.
-
- [1] Recent developments in the ABINIT software package.
- Computer Phys. Comm. 205, 106 (2016).
- X.Gonze, F.Jollet, F.Abreu Araujo, D.Adams, B.Amadon, T.Applencourt,
- C.Audouze, J.-M.Beuken, J.Bieder, A.Bokhanchuk, E.Bousquet, F.Bruneval
- D.Caliste, M.Cote, F.Dahm, F.Da Pieve, M.Delaveau, M.Di Gennaro,
- B.Dorado, C.Espejo, G.Geneste, L.Genovese, A.Gerossier, M.Giantomassi,
- Y.Gillet, D.R.Hamann, L.He, G.Jomard, J.Laflamme Janssen, S.Le Roux,
- A.Levitt, A.Lherbier, F.Liu, I.Lukacevic, A.Martin, C.Martins,
- M.J.T.Oliveira, S.Ponce, Y.Pouillon, T.Rangel, G.-M.Rignanese,
- A.H.Romero, B.Rousseau, O.Rubel, A.A.Shukri, M.Stankovski, M.Torrent,
- M.J.Van Setten, B.Van Troeye, M.J.Verstraete, D.Waroquier, J.Wiktor,
- B.Xue, A.Zhou, J.W.Zwanziger.
- Comment : the fourth generic paper describing the ABINIT project.
- Note that a version of this paper, that is not formatted for Computer Phys. Comm. 
- is available at http://www.abinit.org/about/ABINIT16.pdf .
- The licence allows the authors to put it on the Web.
-
- [2] ABINIT : First-principles approach of materials and nanosystem properties.
- Computer Phys. Comm. 180, 2582-2615 (2009).
- X. Gonze, B. Amadon, P.-M. Anglade, J.-M. Beuken, F. Bottin, P. Boulanger, F. Bruneval,
- D. Caliste, R. Caracas, M. Cote, T. Deutsch, L. Genovese, Ph. Ghosez, M. Giantomassi
- S. Goedecker, D.R. Hamann, P. Hermet, F. Jollet, G. Jomard, S. Leroux, M. Mancini, S. Mazevet,
- M.J.T. Oliveira, G. Onida, Y. Pouillon, T. Rangel, G.-M. Rignanese, D. Sangalli, R. Shaltaf,
- M. Torrent, M.J. Verstraete, G. Zerah, J.W. Zwanziger
- Comment : the third generic paper describing the ABINIT project.
- Note that a version of this paper, that is not formatted for Computer Phys. Comm. 
- is available at http://www.abinit.org/about/ABINIT_CPC_v10.pdf .
- The licence allows the authors to put it on the Web.
-
- [3] A brief introduction to the ABINIT software package.
- Z. Kristallogr. 220, 558-562 (2005).
- X. Gonze, G.-M. Rignanese, M. Verstraete, J.-M. Beuken, Y. Pouillon, R. Caracas, F. Jollet,
- M. Torrent, G. Zerah, M. Mikami, Ph. Ghosez, M. Veithen, J.-Y. Raty, V. Olevano, F. Bruneval,
- L. Reining, R. Godby, G. Onida, D.R. Hamann, and D.C. Allan.
- Comment : the second generic paper describing the ABINIT project. Note that this paper
- should be cited especially if you are using the GW part of ABINIT, as several authors
- of this part are not in the list of authors of the first or third paper.
- The .pdf of the latter paper is available at http://www.abinit.org/about/zfk_0505-06_558-562.pdf.
- Note that it should not redistributed (Copyright by Oldenburg Wissenshaftverlag,
- the licence allows the authors to put it on the Web).
-
- And optionally:
-
- [4] First-principles computation of material properties : the ABINIT software project. 
- X. Gonze, J.-M. Beuken, R. Caracas, F. Detraux, M. Fuchs, G.-M. Rignanese, L. Sindic,
- M. Verstraete, G. Zerah, F. Jollet, M. Torrent, A. Roy, M. Mikami, Ph. Ghosez, J.-Y. Raty, D.C. Allan.
- Computational Materials Science 25, 478-492 (2002). http://dx.doi.org/10.1016/S0927-0256(02)00325-7
- Comment : the original paper describing the ABINIT project.
-
- [5] Towards a potential-based conjugate gradient algorithm for order-N self-consistent
- total energy calculations.
- X. Gonze, Phys. Rev. B 54, 4383 (1996).
- Comment : The potential-based conjugate-gradient algorithm, used when iscf=5, is not published.
- However, many elements of this algorithm have been explained in the paper above.
-
- Proc.   0 individual time (sec): cpu=          0.6  wall=          0.6

================================================================================

 Calculation completed.
.Delivered   2 WARNINGs and   6 COMMENTs to log file.
+Overall time at end (sec) : cpu=          0.6  wall=          0.6
